packages/scripts/README.md

```diff
@@ -18,14 +18,16 @@
 - [Website][website] ([source][website-source])
 - [Documentation][documentation]
 - [Guide][guide] ([source][guide-source])
-  See also the [Update Guide][guide-update], including updated and removed items in the library.
+  Also see the v13 to v14 [Update Guide][guide-update], which includes updated and removed items from the library.
 - [discord.js Discord server][discord]
 - [Discord API Discord server][discord-api]
 - [GitHub][source]
 - [Related libraries][related-libs]
 
 ## Contributing
 
+Before creating an issue, please ensure that it hasn't already been reported/suggested, and double-check the
+[documentation][documentation].  
```

Comment:
This is on 2 lines? Make it 1.

Also, for all subpackages, change the link to `https://discordjs.dev/docs/packages/{package}`. For apps/, make it https://discordjs.dev.

Reply:
Some packages don't have a `https://discordjs.dev/` page, what should we do with them?

Reply:
Don't include them. You said these documents are "out of sync", but who said they should have been in sync in the first place?

> the readme should be highly relevant to each package.

Reply:
I meant the inconsistency in about (as in some don't have them, some do), contributing and the help section. I do not mean with "in sync" that they have to be exactly the same in their description, examples, etc..